apps/documents/views.py

```diff
@@ -349,3 +349,49 @@ def form_valid(self, form):
             assistant_id=assistant.id,
         )
         return HttpResponseRedirect(self.get_success_url())
+
+
+class FileChunkEmbeddingListView(LoginAndTeamRequiredMixin, ListView, PermissionRequiredMixin):
+    """View to display file chunks for a specific file in a collection with pagination"""
+
+    model = FileChunkEmbedding
+    template_name = "documents/file_chunks.html"
+    context_object_name = "chunks"
+    paginate_by = 10
+    permission_required = ("documents.view_collection", "files.view_file")
+
+    def get_queryset(self):
+        collection_id = self.kwargs["collection_id"]
+        file_id = self.kwargs["file_id"]
+
+        # Get chunks for this file in this collection, ordered by chunk number
+        return FileChunkEmbedding.objects.filter(
+            collection_id=collection_id, file_id=file_id, is_archived=False
+        ).order_by("chunk_number")
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        team_slug = self.kwargs["team_slug"]
+        collection_id = self.kwargs["collection_id"]
+        file_id = self.kwargs["file_id"]
+
+        collection_file = get_object_or_404(
+            CollectionFile.objects.select_related("file", "collection"),
```

Comment:
these `select_related` don't seem necessary for this view

Reply:
We're injecting the `collection`, `file` into the context below that, but we get them from this object